app/src/interfaces/input-rich-text-html/useSourceCode.ts

```diff
@@ -20,7 +20,7 @@ export default function useSourceCode(editor: Ref<any>): UsableSourceCode {
 	const code = ref<string>();
 
 	const sourceCodeButton = {
-		icon: 'sourcecode',
+		icon: 'curly-brackets',
```

Comment:
See the other comment

```suggestion
		icon: 'edit-code',
```